pyaerocom/helpers.py

```diff
@@ -806,7 +806,7 @@ def calc_climatology(s, start, stop, min_count=None, set_year=None, resample_how
 
 
 def resample_errors(ds: pd.Series, *args, **kwargs):
-    return math.sqrt((ds**2).sum(skipna=True))
+    return math.sqrt((ds**2).sum(skipna=True)) / len(ds)
```

Comment:
close, but `len(ds)` does not `skipna` in case of `nan` values

Reply:
and maybe, we should make sure that values which are `nan` have errors which are `nan`.